### PR TITLE
register block as mesh object

### PR DESCRIPTION
**Pull request by GeneKao**
### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [x] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### CODE REVIEW

File: src/compas_view2/objects/__init__.py [hunk 1]
```diff
@@ -33,3 +40,6 @@
 Object.register(Sphere, SphereObject)
 Object.register(Torus, TorusObject)
 Object.register(Arrow, ArrowObject)
+
+if is_assembly:
+    Object.register(Block, MeshObject)
```
Comment (brgcode) [inline src/compas_view2/objects/__init__.py:45]: why not just put the try-except here?

Comment (GeneKao) [reply src/compas_view2/objects/__init__.py]: ok @brgcode, changed in commit 7d9b35db05acf1bf430d285fa14acb7cbf765a07

File: src/compas_view2/objects/__init__.py [hunk 2]
```diff
@@ -33,3 +39,8 @@
 Object.register(Sphere, SphereObject)
 Object.register(Torus, TorusObject)
 Object.register(Arrow, ArrowObject)
+
+try:
+    Object.register(Block, MeshObject)
+except:
+    pass
```
Comment (brgcode) [inline src/compas_view2/objects/__init__.py:46]:
```python
try:
    from compas_assembly.datastructures import Block
except ModuleNotFoundError:
    pass
else:
    Object.register(Block, MeshObject)
```

Comment (GeneKao) [reply src/compas_view2/objects/__init__.py]: @brgcode changed in ef3b803b5c8d8b6347b1532a6d5db78dde8f19ee

**Pull request merged**